server/github_mock.go

```diff
@@ -0,0 +1,50 @@
+// +build mock
+
+package server
+
+import (
+	"net/http"
+	"strconv"
+	"time"
+)
+
+const mockServerAddr = "127.0.0.1:8885"
```

Comment:
The mock GitHub server would be better to listen on any available address instead of this arbitrary one, or, as I mentioned previously, not mocking GitHub altogether. Not a big deal, let's leave it as is for now.